### PR TITLE
add box as adopter

### PR DESCRIPTION
**Pull request by itsmurugappan**
Add Box as an adopter. Change Murugappan company to Box.

### CODE REVIEW

File: ADOPTERS.MD [hunk 1]
```diff
@@ -33,5 +33,6 @@ If you would like to be included in this table, please submit a comment to [this
 | VMware                             | Cloud Native Runtimes for VMware Tanzu                                 | Generally Available      | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware                             | VMware Event Broker Appliance                                          | Development              | [link](https://vmweventbroker.io/)                                   |
 | vorteil pty ltd*                   | Serverless orchestration engine                                        | Development & Production | [link](http://vorteil.io)                                            |
+| Box* | | Development | [link](https://box.com/) |
```
Comment (csantanapr) [inline ADOPTERS.MD:36]: Put at the top in order

Comment (csantanapr) [reply ADOPTERS.MD]: Also a short description of the product like "Box SaaS"

Comment (csantanapr) [reply ADOPTERS.MD]: I mean for the column "Product/Project Name"

Comment (itsmurugappan) [reply ADOPTERS.MD]: done.

**Comment by csantanapr** [comment]
Thank you @itsmurugappan I'm very happy to continue being part of SC
/lgtm

**Pull request merged**